pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>11</java.version>
+        <java.version>21</java.version>
```

Comment:
This version determines the source/target version of our Java project, and should actually stay the same. It determines the minimum Java version that our clients need to use if they want to depend on our model(s), so upgrading this one is backwards incompatible.

For now, we only need to upgrade our Java build version. (so what you have done in CodeFresh)